### ISSUE TITLE
Add JSON-LD Frame as informative annex

### ISSUE THREAD

**Issue by azaroth42** [w3c/web-annotation#80]
@azaroth42 or @paolociccarese to provide

**Comment by azaroth42** [comment]
Ooops. Still to do for next Model WD. :blush:

**Comment by azaroth42** [comment]
Frames needed:
- [x] Annotation frame
- [x] Collection frame
- [x] Collection Page frame, embedded URIs
- [x] Collection Page frame, embedded Annotations

Any others?

**Comment by azaroth42** [comment]
Blocked:  https://lists.w3.org/Archives/Public/public-linked-json/2016Feb/0008.html
Propose `postpone`.

**Comment by iherman** [comment]
+1

**Comment by azaroth42** [comment]
Unblocked if we change the way the alias for @type is handled in the context document to refer directly to @type, rather than all the way down to rdf:type.  (per JSON-LD list, verified locally)  Will make the change.

**Comment by azaroth42** [comment]
Done: http://w3c.github.io/web-annotation/vocab/wd/#json-ld-frames
